r/tests/testthat/test-s3-minio.R

```diff
@@ -54,6 +54,24 @@ if (arrow_with_s3() && process_is_running("minio server")) {
     )
   })
 
+  test_that("read/write compressed csv by filesystem", {
+    dat <- tibble(x = seq(1, 10, by = 0.2))
```

Comment:
Does this need to be skipped if gzip is not present or have we already skipped by this point?